server/models/announcements.ts

```diff
@@ -0,0 +1,24 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../database/connection';
+import Classes from './classes';
+
+const Announcements = sequelize.define('Announcement', {
```

Comment:
Usually, models have singular names and tables have pluralized names, so change all names.